examples/bpk-component-section-header/stories.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import {
+  DefaultExample,
+  WithDescriptionExample,
+  WithButtonExample,
+  FullExample,
+  MobileExample,
+} from './examples';
+
+export default {
+  title: 'bpk-component-section-header',
+};
+
+export const Default = DefaultExample;
+
+export const WithDescription = WithDescriptionExample;
+
+export const WithButton = WithButtonExample;
+
+export const FullProps = FullExample;
+
+export const ForMobile = MobileExample;
+
+export const VisualTestDefault = DefaultExample;
```

Comment:
No need to have a visual test for each of these. Instead you can bundle them all into one example component that gets visually tested. We do this in most of our components for example - https://github.com/Skyscanner/backpack/blob/main/examples/bpk-component-badge/examples.js#L140